GrpcClient/EmployeeGrpcRepository.cs

```diff
@@ -0,0 +1,286 @@
+        var reply = await client.IsEmployeeInRepositoryAsync(new Id { Id_ = Id });
+        b = reply.Boolean_;
+        
+        return b;
+    }
+
+    public static List<Entities.Shift> GrpcShiftDtosToEntityShiftsList(List<ShiftDTO> grpcShiftDtos)
+    {
+        List<Entities.Shift> shifts = new List<Entities.Shift>();
+        foreach (var shiftDTO in grpcShiftDtos)
+        {
+            var shift = new Entities.Shift
+            {
+                Id = shiftDTO.Id,
+                StartDateTime = new DateTime(shiftDTO.StartDateTime),
+                EndDateTime = new DateTime(shiftDTO.EndDateTime),
+                TypeOfShift = shiftDTO.TypeOfShift,
+                ShiftStatus = shiftDTO.ShiftStatus,
+                Description = shiftDTO.Description,
+                Location = shiftDTO.Location,
+                AssingnedEmployees = shiftDTO.AssignedEmployeeIds.ToList()
+            };
+            shifts.Add(shift);
+        }
+
+        return shifts;
+    }
+    
+    public static List<Entities.Shift> EntityShiftDtosToEntityShiftsList(List<DTOs.Shift.ShiftDTO> shiftDtos)
+    {
+        List<Entities.Shift> shifts = new List<Entities.Shift>();
+        foreach (var shiftDto in shiftDtos)
+        {
+            var shift = new Entities.Shift
+            {
+                Id = shiftDto.Id,
+                StartDateTime = shiftDto.StartDateTime,
+                EndDateTime = shiftDto.EndDateTime,
+                TypeOfShift = shiftDto.TypeOfShift,
+                ShiftStatus = shiftDto.ShiftStatus,
+                Description = shiftDto.Description,
+                Location = shiftDto.Location,
+                AssingnedEmployees = shiftDto.AssignedEmployees
+            };
+            shifts.Add(shift);
+        }
+
+        return shifts;
+    }
+
+    public static Entities.Employee grpcEmployeeDtoToEntityEmployee(EmployeeDTO employeeDto, List<Entities.Shift> shifts)
+    {
+        Entities.Employee employee = new Entities.Employee
+        {
+            FirstName = employeeDto.FirstName,
+            LastName = employeeDto.LastName,
+            Id = employeeDto.Id,
+            WorkingNumber = int.CreateChecked(employeeDto.WorkingNumber),
+            Shifts = shifts,
+            Email = employeeDto.Email,
+            Password = employeeDto.Password,
+        };
+        return employee;
+    }
+    
+    public static List<Entities.Employee>grpcEmployeeDtoListToEntityEmployeeList(EmployeeDTOList list)
+    {
+        List<Entities.Employee> employees = new List<Entities.Employee>();
+        foreach (var dto in list.Dtos)
+        {
+            Entities.Employee employee = new Entities.Employee
+            {
+                FirstName = dto.FirstName,
+                LastName = dto.LastName,
+                Id = dto.Id,
+                WorkingNumber = int.CreateChecked(dto.WorkingNumber),
+                Shifts = GrpcShiftDtosToEntityShiftsList(dto.AssignedShifts.Dtos.ToList()),
```

Comment:
Why no just Shifts = shifts like in previous method?

Reply:
it would not let it, because its in different datatype so we r using this long-ass converter we made for it